Jenkinsfile.release

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+ansiColor('xterm') {
+  node('JenkinsMarathonCI-Debian8-2017-10-23') {
+
+    properties([
+      parameters([
+        string(name: 'gitsha',
+          defaultValue: '',
+          description: 'Git Commit SHA to Build and Release (ex. 167d7fb86 or 167)'
+        ),
+        string(name: 'version',
+          defaultValue: '',
+          description: 'Community Release Version (ex. v1.5.2)'
+        ),
+        string(name: 'latest',
+          defaultValue: 'False',
+          description: 'True if docker mesosphere/marathon:latest should be pushed'
```

Comment:
🤔 in which cases would this be false?

Reply:
Today when we release 1.5.2 it will be True.   Tomorrow when we release 1.6.0 it will be True.   in 3 days when release 1.5.3, it will be False.

Reply:
Ah, that makes a lot of sense. Thanks.